apps/algolia/lib/algolia/api.ex

```diff
@@ -8,6 +8,8 @@ defmodule Algolia.Api do
 
   defstruct [:host, :index, :action, :body]
 
+  @type action :: :post | :delete
+
```

Comment:
Changes to this file aren't required for the code change.  I modified this file under the impression that I would have to make other network calls outside of `POST`.  I decided to leave it in as it allows us to easily add support for `GET` in the future if we need it

Reply:
I like the sentiment but this implementation strikes me as incomplete so I'm leaning towards removing. It's a good idea, if we do change this to make other types of Algolia REST API calls it should be changed more substantively to include more verbs and cover more endpoints.